source/docs/software/advanced-controls/filters/slew-rate-limiter.rst

```diff
@@ -82,7 +82,7 @@ A typical use of a SlewRateLimiter is to limit the acceleration of a robot's dri
     drivetrain.ArcadeDrive(forward, turn);
 
     // Slew-rate limits the forward/backward input, limiting forward/backward acceleration
-    drivetrain.ArcadeDrive(filter.Calculate(forward), turn);
+    drivetrain.ArcadeDrive(filter.Calculate(units::voltage::volt_t{forward}).value(), turn);
```

Comment:
```suggestion
    drivetrain.ArcadeDrive(filter.Calculate(forward).value(), turn);
```

Reply:
If you are getting the input in most cases it should be a units::volt_t as that is the correct way to do things in C++ so it should be like this